frontend/src/components/GoogleMapsContainer.js

```diff
@@ -27,7 +27,7 @@ export default function GoogleMapsContainer({locations}) {
         <Wrapper>
             <GoogleMap
                 mapContainerStyle={mapContainerStyle}
-                zoom={5}
+                zoom={9}
                 center={mapCenter}>
                 {locations.map((location) =>
                     <LocationMarker key={location.id} position={{lat: location.lat, lng: location.lng}} location={location}/>
```

Comment:
Why pass explicitly create a position prop and object, when lat and long are already passed in via location?